pkg/projectfile/projectfile.go

```diff
@@ -814,13 +821,18 @@ func GetProjectFilePath() (string, error) {
 	return "", &ErrorNoProject{locale.NewInputError("err_no_projectfile")}
 }
 
+var printedActiveStateProjectDeprecationNotice = false
+
 func getProjectFilePathFromEnv() (string, error) {
 	var projectFilePath string
 
+	if legacy := os.Getenv("ACTIVESTATE_PROJECT"); legacy != "" && !printedActiveStateProjectDeprecationNotice {
+		out.Notice(locale.T("warning_activestate_project_env_var"))
+		printedActiveStateProjectDeprecationNotice = true
+	}
```

Comment:
projectfile should not be dealing with output. I think we could put this in main. Maybe create a `deprecation.go` file where we can place warnings like this.